pygame_menu/menu.py

```diff
@@ -58,7 +58,7 @@ class Menu(object):
     :type width: int, float
     :param title: Title of the Menu (main title)
     :type title: str
-    :parama auto_center_v: Auto centers the menu on the vertical position after a widget is added/deleted
```

Comment:
I noticed that when I changed this from "parama" to "param", it didn't pick up on this parameter. Has it been removed?

Reply:
Yes this has been removed,

Reply:
This should be `center_content`

Reply:
Okay. I'll change it to match.